### PR TITLE
Elasticsearch: only sort on properties know to be in the index

### PR DESCRIPTION
**Pull request by joeferner**
@sfeng88 @srfarley

### CODE REVIEW

File: test/src/main/java/org/vertexium/test/GraphTestBase.java [hunk 1]
```diff
@@ -2085,6 +2085,23 @@ public void testGraphQuerySort() {
     }
 
     @Test
+    public void testGraphQuerySortThatHasntBeenTested() {
```
Comment (srfarley) [inline test/src/main/java/org/vertexium/test/GraphTestBase.java]: Did you mean to call this `testGraphQuerySortThatHasntBeenIndexed()`?

Comment (joeferner) [inline test/src/main/java/org/vertexium/test/GraphTestBase.java]: fixed

**Comment by sfeng88** [comment]
lgtm

**Comment by srfarley** [comment]
👍

**Pull request merged**